src/Dfc.CourseDirectory.Core/DataManagement/IFileUploadProcessor.cs

```diff
@@ -21,5 +21,6 @@ public interface IFileUploadProcessor
         Task<UploadStatus> WaitForVenueProcessingToCompleteForProvider(Guid providerId, CancellationToken cancellationToken);
 
         Task<SaveFileResult> SaveCourseFileForProvider(Guid providerId, Stream stream, UserInfo uploadedBy);
+        Task<bool> DeleteVenueUploadRow(Guid venueUploadId, int rowNumber);
```

Comment:
Nit: move into the section with other venue bits

Reply:
done